cl/recap/tasks.py

```diff
@@ -1658,11 +1606,8 @@ def process_recap_email(
             recap_email_recipients = get_recap_email_recipients(
                 data["email_recipients"]
             )
-            chain(
-                send_docket_alert.si(
-                    docket.pk, start_time, recap_email_recipients
-                ),
-                send_docket_to_webhook.si(docket.pk, start_time, epq.pk),
+            send_docket_alert.si(
+                docket.pk, start_time, recap_email_recipients
```

Comment:
If we're not doing a chain and don't need special stuff provided by apply_async, we should just use .delay().

Reply:
changed!

Reply:
There's an even better way to do this. Check out the example here, which drops the `si` too: https://docs.celeryq.dev/en/stable/userguide/calling.html#basics

Reply:
Thanks, I've changed it.